engine-tests/src/tests/eth_connector.rs

```diff
@@ -29,6 +28,18 @@ const EVM_CUSTODIAN_ADDRESS: &'static str = "096DE9C2B8A5B8c22cEe3289B101f6960d6
 const DEPOSITED_EVM_AMOUNT: u128 = 10200;
 const DEPOSITED_EVM_FEE: u128 = 200;
 
+#[derive(
+    Default, BorshDeserialize, BorshSerialize, Clone, serde::Deserialize, serde::Serialize,
+)]
+pub struct Proof {
```

Comment:
Previously `Proof` was imported. Is this duplicating the struct now? I think it is better to avoid duplication.

Reply:
It's another interesting issue:
```
#[derive(Default, BorshDeserialize, BorshSerialize, Clone)]
#[cfg_attr(test, derive(serde::Deserialize, serde::Serialize))]
pub struct Proof {
```
So after moving to a new crate:
`#[cfg_attr(test, derive(serde::Deserialize, serde::Serialize))]`
doesn't work. Thay why  code duplicated.

Reply:
Ah, I see. So the `serde` serialization is the reason. We do not use this in production, but we do use this in the tests.

As far as I remember the reason for this is simply convenience, so we could refactor the tests to not need `serde`. Perhaps that is out of scope for this PR though.